src/libraries/System.Private.CoreLib/src/System/Math.DivModInt.cs

```diff
@@ -0,0 +1,224 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+#if NATIVEAOT
+using System.Runtime.InteropServices;
+#endif
+
+namespace System
+{
+    /// <summary>
+    /// Provides constants and static methods for trigonometric, logarithmic, and other common mathematical functions.
```

Comment:
It feels a bit weird to have documentation of the whole `Math` method on a minor portion of it.

Reply:
It's stating the purpose of this partial, was moved from the deleted MathHelpers.cs. There are 654 partial classes with diffing summaries from their partials in runtime repo. VS also knows how to handle it beased on the context.

Reply:
It's stating the purpose of this partial, was moved from the deleted MathHelpers.cs. There are 654 partial classes with diffing summaries from their partials in runtime repo. VS also knows how to handle it based on the context.

Reply:
> It's stating the purpose of this partial

This file does not have "constants and static methods for trigonometric, logarithmic, and other common mathematical functions".

Reply:
Ah, I had those before you asked me to split the file in the earlier review. I've matched the comment with the MathHelpers.cs.

Reply:
Ah, I had those before you asked me to split the file in the earlier review. I've matched the summary with MathHelpers.cs.